groups/live-windows-virtual-machines/keys.tf

```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------------------------
+# Key Pair
+# ------------------------------------------------------------------------------
+resource "aws_key_pair" "ewok_lic_bac_keypair" {
+  key_name   = "Ewok-License-Backup-Server"
+  public_key = local.ewok_lic_bac_ec2_data["public-key"]
+}
+
+resource "aws_key_pair" "ewok_lic_sec_keypair" {
+  key_name   = "Ewok-License-Second-Server"
+  public_key = local.ewok_lic_sec_ec2_data["public-key"]
+}
```

Comment:
These are in the same application stack so can use the same key pair

Reply:
One key used for both servers.